src/fireblocks-sdk.ts

```diff
@@ -2447,4 +2453,52 @@ export class FireblocksSDK {
         };
         return this.apiClient.issuePostRequest(`/v1/vault/assets/bulk`, body, requestOptions);
     }
+
+    /**
+     * Creates new payment workflow configuration
+     * @param request - Payments.CreateWorkflowConfigurationRequest
+     */
+    public async createPaymentWorkflowConfiguration(request: Payments.CreateWorkflowConfigurationRequest): Promise<Payments.WorkflowConfiguration> {
+        return await this.paymentsApiClient.createPaymentWorkflowConfiguration(request);
+    }
+
+    /**
+     * Get payment workflow configuration
+     * @param configId - configuration id
+     */
+    public async getPaymentWorkflowConfiguration(configId: string): Promise<Payments.WorkflowConfiguration> {
+        return await this.paymentsApiClient.getPaymentWorkflowConfiguration(configId);
+    }
+
+    /**
+     * Delete payment workflow configuration
+     * @param configId - configuration id
+     */
+    public async deletePaymentWorkflowConfiguration(configId: string): Promise<Payments.WorkflowConfigurationId> {
+        return await this.paymentsApiClient.deletePaymentWorkflowConfiguration(configId);
+    }
+
+    /**
+     * Creates new payment workflow execution
+     * @param request - Payments.CreateWorkflowExecutionRequest
+     */
+    public async createPaymentWorkflowExecution(request: Payments.CreateWorkflowExecutionRequest): Promise<Payments.WorkflowExecution> {
+        return await this.paymentsApiClient.createPaymentWorkflowExecution(request);
+    }
+
+    /**
+     * Get payment workflow execution
+     * @param workflowExeuctionId
+     */
+    public async getPaymentWorkflowExecution(workflowExeuctionId: string): Promise<Payments.WorkflowExecution> {
+        return await this.paymentsApiClient.getPaymentWorkflowExecution(workflowExeuctionId);
+    }
+
+    /**
+     * Execute payment workflow
+     * @param workflowExeuctionId
+     */
+    public async executePaymentFlow(workflowExeuctionId: string): Promise<Payments.WorkflowExecution> {
```

Comment:
Just to keep the same pattern, this is the only endpoint with the word `Flow` instead of `Workflow`.
I guess it came from our code but maybe it's better to change it here.